### PR TITLE
daemon/delete.go: use less confusing naming of functions

### PR DESCRIPTION
**Pull request by LK4D4**
ping @duglin

### CODE REVIEW

**Comment by duglin** [comment]
design LGTM - you da man! thanks!  :-)

File: daemon/delete.go [hunk 1]
```diff
@@ -80,8 +56,38 @@ func (daemon *Daemon) ContainerRm(name string, config *ContainerRmConfig) error
...
+		if err := daemon.updateNetwork(parentContainer); err != nil {
+			logrus.Debugf("Could not update network to remove link %s: %v", n, err)
+		}
+	}
+
+	return nil
+}
+
+// cleanupContainer unregisters a container from the daemon, stops stats
+// collectionand cleanly removes contents and metadata from the filesystem.
```
Comment (MHBauer) [inline daemon/delete.go]: space between "collection" and "and"

**Comment by MHBauer** [comment]
nit, looks fine otherwise.

**Comment by cpuguy83** [comment]
LGTM pending janky, but also can't merge because it says it's out of date from master.

**Comment by calavera** [comment]
LGTM

**Pull request merged**